test/sourcemaps/samples/external/test.js

```diff
@@ -0,0 +1,27 @@
+import { getLocator } from 'locate-character';
+import { COMMON, STYLES } from './_config';
+
+export function test({ assert, input, preprocessed }) {
+
+	const assertMapped = (locateInput, code, filename) => {
```

Comment:
I really like the `asserMapped` and the `assertUnmapped` function. Could this be something to extract into test utils? They look the same across the test files.